spec/models/vacancy_spec.rb

```diff
@@ -325,8 +325,8 @@
           expired = build(:vacancy, :expired)
           expired.send :set_slug
           expired.save(validate: false)
-          expires_today = create(:vacancy, expires_on: Time.zone.today)
-          expires_future = create(:vacancy, expires_on: 3.months.from_now)
+          expires_today = create(:vacancy, :with_no_expiry_time, expires_on: Time.zone.today)
```

Comment:
`with_no_expiry_time` doesn't set the expiry time but maybe we could rename this.
I could be wrong, but it feels a bit misleading to say it has no expiry time.
Maybe `set_expiry_at_midnight`? or something similar?

Reply:
Not sure whats a better name, this referring to vacancies where expiry_time comumn is nil, therefore `set_expiry_at_midnight` feels a bit misleading to me

Reply:
`with_no_expiry_time` explicitly sets the expiry time to `nil`, we just treat such values as the end of the day when handling them in the code